### PR TITLE
Bug 1522761 Add `K8S_HOST_URL` Variable

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1522761

### CODE REVIEW

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../install_config/aggregate_logging.adoc#install-config-aggregate-logging[Aggregating Container Logs]
|Added the `openshift_logging_fluentd_master_url` variable to the
xref:../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Specifying Logging Ansible Variables] section.
%

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL 🌞

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -338,6 +338,10 @@ audit log of the container engine is collected and stored in ES.
 |Location of the Fluentd `in_tail` position file for the audit log file. The default is
 `/var/log/audit/audit.log.pos`.
 
+|`openshift_logging_fluentd_master_url`
+|In Fluend, `K8S_HOST_URL` is an internal url for reaching the master API to
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: s/url/URL

**Comment by ahardin-rh** [comment]
@bmcelvee Just one minor comment from me. 👍

**Pull request closed without merging**